diff --git a/setup.py b/setup.py
index 7c47082..bf07f64 100755
--- a/setup.py
+++ b/setup.py
@@ -128,11 +128,12 @@
             r"python/cv2/.*config.*.py"
         ]
         +
-        [ r"python/cv2/py.typed" ] if sys.version_info >= (3, 6) else []
+        # https://github.com/opencv/opencv-python/issues/871
+        #[ r"python/cv2/py.typed" ] if sys.version_info >= (3, 6) else []
+        []
         ,
-        "cv2.data": [  # OPENCV_OTHER_INSTALL_PATH
-            ("etc" if os.name == "nt" else "share/opencv4") + r"/haarcascades/.*\.xml"
-        ],
+        # https://github.com/opencv/opencv-python/issues/871
+        # "cv2.data" entry removed to avoid haarcascades file not found error
         "cv2.gapi": [
             "python/cv2" + r"/gapi/.*\.py"
         ],
@@ -147,8 +148,9 @@
         ],
     }
 
-    if sys.version_info >= (3, 6):
-        rearrange_cmake_output_data["cv2.typing"] = ["python/cv2" + r"/typing/.*\.py"]
+    # https://github.com/opencv/opencv-python/issues/871
+    #if sys.version_info >= (3, 6):
+    #    rearrange_cmake_output_data["cv2.typing"] = ["python/cv2" + r"/typing/.*\.py"]
 
     # Files in sourcetree outside package dir that should be copied to package.
     # Raw paths relative to sourcetree root.
@@ -500,7 +502,7 @@
 
     # generate a fresh version.py always when Git repository exists
     # (in sdists the version.py file already exists)
-    if os.path.exists(".git"):
+    if False: #os.path.exists(".git"):
         old_args = sys.argv.copy()
         sys.argv = ["", str(contrib), str(headless), str(rolling), str(ci_build)]
         runpy.run_path("find_version.py", run_name="__main__")
